pmd-java/src/test/java/net/sourceforge/pmd/typeresolution/ClassTypeResolverTest.java

```diff
@@ -270,8 +272,7 @@ public void testAnonymousInnerClass() throws ClassNotFoundException {
 
     @Test
     public void testLiterals() throws JaxenException {
-        ASTCompilationUnit acu = parseAndTypeResolveForClass15(Literals.class);
-        List<ASTLiteral> literals = convertList(acu.findChildNodesWithXPath("//Literal"), ASTLiteral.class);
+        List<ASTLiteral> literals = selectNodes(Literals.class, ASTLiteral.class, "//Literal");
```

Comment:
this xpath selector could be scrapped, right?

Reply:
Good catch